package.json

```diff
@@ -53,15 +53,15 @@
     "@nuxt/typescript-runtime": "^2.1.0",
     "core-js": "^3.15.1",
     "matter-zk-ui": "^1.0.9",
-    "nuxt": "^2.15.7"
+    "nuxt": "latest"
```

Comment:
it's okay use latest tag for this package?
fyi @andriidrebot

Reply:
Yeah, I don't think this is ok either